js/utils/dom.js

```diff
@@ -19,3 +19,33 @@ export function isScrolledIntoView(element) {
   // Check its within the document viewport
   return top <= document.documentElement.clientHeight;
 }
+
+// http://stackoverflow.com/a/41426040/632806
+export function insertAtCursor(myField, myValue) {
+  if (document.selection) {
```

Comment:
I'm thinking it might be wiser to leave it in so this is more portable. Especially, since this is a generic utility function, it's very likely we'd use it in our future web-app and it seems easier to have the cross-browser stuff there rather than have to hunt it down later.

I wouldn't of made an effort to put the cross browser stuff in, but since it came for free, I brought it over.